wagtail_localize/views/edit_translation.py

```diff
@@ -971,14 +893,10 @@ def get_translation_progress(segment, locale):
     else:
         add_convert_to_alias_url = False
 
-    if WAGTAIL_VERSION >= (4, 0):
-        has_legacy_styling = False
-        side_panels = (
-            LocalizedPageSidePanels(request, instance, translation) if is_page else None
-        )
-    else:
-        has_legacy_styling = True
-        side_panels = None
+    has_legacy_styling = False
```

Comment:
`has_legacy_styling` refers to Wagtail 2.x, so can be dropped

Reply:
Removed all instances of `has_legacy_styling`